tests/test_acl_portchannel.py

```diff
@@ -0,0 +1,190 @@
+            if fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_GROUP_ID":
+                assert table_group_id == fv[1]
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_ID":
+                table_id = fv[1]
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY":
+                assert fv[1] == "100"
+            else:
+                assert False
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE")
+        (status, fvs) = tbl.get(table_id)
+        assert status == True
+
+        # remove ACL table
+        self.remove_acl_table(dvs, "LAG_ACL_TABLE")
+
+        # remove port channel
+        self.remove_port_channel(dvs, "PortChannel01")
+
+    # Frist create ACL table
+    # Second create port channel
+    def test_PortChannelAcl2(self, dvs):
+        self.setup_db(dvs)
+
+        # create ACL table
+        self.create_acl_table(dvs, "LAG_ACL_TABLE", "PortChannel01")
+
+        # create port channel
+        self.create_port_channel(dvs, "PortChannel01")
+
+        # check ASIC table
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_LAG")
+        lag = tbl.getKeys()[0]
+        (status, fvs) = tbl.get(lag)
+        assert status == True
+        assert len(fvs) == 2
+        for fv in fvs:
+            if fv[0] == "SAI_LAG_ATTR_INGRESS_ACL":
+                table_group_id = fv[1]
+            elif fv[0] == "NULL":
+                continue
+            else:
+                assert False
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+        (status, fvs) = tbl.get(table_group_id)
+        assert status == True
+        assert len(fvs) == 3
+        for fv in fvs:
+            if fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_ACL_STAGE":
+                assert fv[1] == "SAI_ACL_STAGE_INGRESS"
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_TYPE_LIST":
+                assert fv[1] == "1:SAI_ACL_BIND_POINT_TYPE_LAG"
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_ATTR_TYPE":
+                assert fv[1] == "SAI_ACL_TABLE_GROUP_TYPE_PARALLEL"
+            else:
+                assert False
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP_MEMBER")
+        member = tbl.getKeys()[0]
+        (status, fvs) = tbl.get(member)
+        assert status == True
+        assert len(fvs) == 3
+        for fv in fvs:
+            if fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_GROUP_ID":
+                assert table_group_id == fv[1]
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_ID":
+                table_id = fv[1]
+            elif fv[0] == "SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY":
+                assert fv[1] == "100"
+            else:
+                assert False
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE")
+        (status, fvs) = tbl.get(table_id)
+        assert status == True
```

Comment:
check asic db is duplicated. can you share the code with the previous function. need to avoid code duplication.